CHANGELOG.rst

```diff
@@ -4,6 +4,25 @@ Changelog
 (Unreleased)
 ~~~~~~~~~~~~
 
+* **Backwards incompatible:** Tag slugification used to silently strip non-ASCII characters
+ from the tag name to make the slug. This leads to a lot of confusion for anyone using
+ languages with non-latin alphabets, as well as weird performance issues.
+
+ Tag slugification will now, by default, maintain unicode characters as-is during
+ slugification. This will lead to less surprises, but might cause issues for you if you are
+ expecting all of your tag slugs to fit within a regex like `[a-zA-Z0-9]` (for example in
+ URL routing configurations).
+
+ Generally speaking, this should not require action on your part as a library user, as
+ existing tag slugs are persisted in the database, and only new tags will receive the
+ enhanced unicode-compatible slug.
+
+ If you wish to maintain the old stripping behavior, set the setting
+ `TAGGIT_STRIP_UNICODE__WHEN_SLUGIFYING` to `True`.
+
+ As a reminder, custom tag models can easily customize slugification behavior by overriding
+ the `slugify` method to your business needs.
```

Comment:
I will be very helpful to add description `TAGGIT_STRIP_UNICODE__WHEN_SLUGIFYING` in the document as well.
e.g. https://github.com/jazzband/django-taggit/blob/master/docs/getting_started.rst .  in the `Note` section.

Good to also mention that there is another option: install `unidecode` (translate all non ascii character to ascii first). These information is only address in [changelog](https://django-taggit.readthedocs.io/en/latest/changelog.html#id43)

https://github.com/jazzband/django-taggit/blob/2626e235a65f510e1282bccc1a2f7568506642a3/taggit/models.py#L10-L16


```
>>> unidecode.unidecode('あい うえお')
'ai ueo'
```